### ISSUE TITLE
Help pages in tabs

### ISSUE THREAD

**Issue by smichel17** [mpcjanssen/simpletask-android#553]
I think this would be much easier to switch between them. And you wouldn't need to list them at the bottom of the intro page, for example.

**Comment by mpcjanssen** [comment]
One reason to list all the help pages on the main index file is that it allows reading the help on github as well. However changing the help menu items for tabs is a good idea.

**Comment by smichel17** [comment]
I started changing it for tabs but then stopped because it seems like a lot of work for little gain. For example, I'm not sure how to make the in-document links work for reading the help both in-app and on the web.

**Comment by mpcjanssen** [comment]
@smichel17 how about using a navigation drawer instead of the menu? That seems more natural.

**Comment by smichel17** [comment]
@mpcjanssen I agree that a nav drawer is the obvious solution. And with an actual navigationdrawerlayout you can literally copy-paste the menu xml into it and it'll just work.

The problem is, the help screen currently has a back arrow in the app bar, not a hamburger menu, so I' wasn't sure how to implement this behavior intuitively.

I do have ideas right now (tl;dr, right nav drawer) but it's relatively low-priority at the moment for me, compared to changes to the main UI.
